pages/common/git-diff.md

```diff
@@ -19,9 +19,9 @@
 
 `git diff 'HEAD@{3 months|weeks|days|hours|seconds ago}'`
 
-- Show only names of changed files since a given commit:
+- Show diff statistics, like files changed, histogram, and total line insertions/deletions:
```

Comment:
```suggestion
- Show changes statistics, like files changed, histogram, and total line insertions/deletions:
```

Reply:
I think using "diff" is clearer

Reply:
But IMO we should use a full word instead of just `diff`.
```suggestion
- Show difference statistics, like files changed, histogram, and total line insertions/deletions:
```
or
```suggestion
- Show differences statistics, like files changed, histogram, and total line insertions/deletions:
```

Reply:
I think "diff" would be much more apt here. For translations, we can use any equivalent word.